tests/queries/0_stateless/01451_normalize_query.sql

```diff
@@ -23,4 +23,15 @@ SELECT normalizeQuery('SELECT 1, ''xyz11''');
 SELECT normalizeQuery('SELECT $doc$VALUE$doc$ xyz11');
 SELECT normalizeQuery('SELECT $doc$VALUE$doc$, xyz11');
 SELECT normalizeQuery('SELECT $doc$VALUE$doc$, ''xyz11''');
-
+SELECT normalizeQuery('1 - 2');
+SELECT normalizeQuery('1, -2, 3');
+SELECT normalizeQuery('-1, -2, 3');
+SELECT normalizeQuery('1 - 2, 3, 4');
+SELECT normalizeQuery('f(-2, 3)');
+SELECT normalizeQuery('[-1, 2, 3]');
+SELECT normalizeQuery('1 + 2');
+SELECT normalizeQuery('1, +2, 3');
+SELECT normalizeQuery('-1, +2, 3');
+SELECT normalizeQuery('1 + 2, 3, 4');
+SELECT normalizeQuery('f(+2, 3)');
+SELECT normalizeQuery('[+1, 2, 3]');
```

Comment:
Can you please also add tests with addition/subtraction somewhere in the middle of the sequence and the end?